app/auth/defra-id/get-scopes.js

```diff
@@ -1,8 +1,12 @@
+const { USER } = require('../scopes')
+
 const getScopes = (roles) => {
-  return roles.map(role => {
+  const defraIdRoles = roles.map(role => {
     const roleData = role.split(':')
     return roleData.length > 1 ? roleData[1] : roleData[0]
   })
+  const ffdRoles = [USER]
+  return [...defraIdRoles, ...ffdRoles]
```

Comment:
Adding this as currently our prototypes are configured to allow anyone in this default user role.

Until we've explored the roles more, will just append the default role to every user.